### ISSUE TITLE
java.lang.NullPointerException: Autocomplete, when method cannot be found for argument name

### ISSUE THREAD

**Issue by medosobh** [wt-io-it/odoo-pycharm-plugin#82]
Plugin Version: 0.6.6
IntelliJ Version: PyCharm 2022.2.3
IntelliJ Build: PC-222.4345.23

<details><summary>java.lang.NullPointerException: Cannot invoke "com.jetbrains.python.psi.PyReferenceExpression.getLastChild()" because "method" is null</summary>

```
java.lang.NullPointerException: Cannot invoke "com.jetbrains.python.psi.PyReferenceExpression.getLastChild()" because "method" is null
	at at.wtioit.intellij.plugins.odoo.OdooRecordPsiElementMatcherUtil.isOdooRecordPsiElement(OdooRecordPsiElementMatcherUtil.java:61)
	at at.wtioit.intellij.plugins.odoo.OdooCompletionContributor.fillCompletionVariants(OdooCompletionContributor.java:48)
	at com.intellij.codeInsight.completion.CompletionService.getVariantsFromContributors(CompletionService.java:76)
	...
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:297)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	...
	at java.base/java.lang.Thread.run(Thread.java:833)
```
</details>

**Comment by ap-wtioit** [comment]
need to release the fix in 501825d532df2f2eeb3ed6b007ec9bbcdbfc76e1

**Comment by ap-wtioit** [comment]
test case is something like this:
```python
my_dict[name]("my<caret>")
```

**Comment by ap-wtioit** [comment]
Fixed in version 0.6.8 of the plugin (pending approval by jetbrains) for those eager to test the new version can also be found here:
https://github.com/wt-io-it/odoo-pycharm-plugin/tree/v0.6.8

**Comment by bigbear3001** [comment]
0.6.8 is now available in the marketplace
